extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -81,40 +105,78 @@ require([
 				}
 			};
 			uiModal.createComponent(createPageModalConfig, function (modal) {
-				nirvana.sendRequest({
-					controller: 'CommunityPageSpecial',
-					method: activeTab.request,
-					data: { mcache: 'writeonly' }, // fixme: temporary debug variable
-					format: 'json',
-					type: 'get'
-				}).then(function (response) {
-					var html = navHtml + mustache.render(templates[activeTab.template], response);
+				getModalTabContentsHtml(tabToActivate).then(function (tabContentHtml) {
+					var html = navHtml + tabContentHtml;
 
 					modal.$content
 						.addClass('ContributorsModule ContributorsModuleModal')
 						.html(html)
-						.find(activeTab.className).children().addClass('active');
+						.find(tabToActivate.className).children().addClass('active');
 
 					modal.show();
+					activeTab = tabToActivate;
 
 					window.activeModal = modal;
 				});
 			});
 		});
 	}
 
+	function switchCommunityModalTab(tabToActivate) {
+		if (tabToActivate === activeTab) {
+			return;
+		}
+
+		getModalNavHtml().then(function (navHtml) {
+			var html;
+
+			// Switch highlight to new tab
+			// fixme: Loading indicator should be via a template.
+			html = navHtml + $.msg('communitypage-modal-loading');
```

Comment:
Since you're using `.html` you need to HTML escape `$.msg('communitypage-modal-loading')`. The message `communitypage-modal-loading` also doesn't seem to be defined in the i18n.php file?

Also, no need to separate the var declaration from the assignment here.